capstone/src/main/java/com/google/sps/service/FilterService.java

```diff
@@ -0,0 +1,35 @@
+package com.google.sps.service;
+
+import java.io.IOException;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory.Builder;
+
+public class FilterService {
+
+    private static final String KEY_PROPERTY = "__key__";
+
+    public static Entity getEntity(String entity, Long id) {
```

Comment:
entity could be named something which differentiates it from sounding like an Entity (which it clearly isnt since its a String, but still isnt clear what it is supposed to be from just the signature). Something like entity_name is easier to understand from just the signature.